doc/virtualization.rst

```diff
@@ -0,0 +1,461 @@
+
+* GPU Passthrough Virtualization
+
+* GPU SR-IOV Virtualization
+
+The first one, GPU Passthrough Virtualization, is a legacy Virtualization
+Technology which allows exclusive access to GPU from under Virtual Machine (VM).
+GPU SR-IOV Virtualization is a new technology available in modern
+Intel GPUs such as Intel® Data Center GPU Flex Series.
+
++-----------------------------------------------+----------------+------------+
+| Intel GPU                                     | GPU Passtrough | GPU SR-IOV |
++===============================================+================+============+
+| gen8-gen11 (such as BDW, SKL, KBL, CFL, etc.) | |check|        | |cross|    |
++-----------------------------------------------+----------------+------------+
+| TGL                                           | |check|        | |cross|    |
++-----------------------------------------------+----------------+------------+
+| DG1                                           | |check|        | |cross|    |
++-----------------------------------------------+----------------+------------+
+| Alchemist                                     | |check|        | |cross|    |
++-----------------------------------------------+----------------+------------+
+| ATS-M                                         | |check|        | |check|    |
++-----------------------------------------------+----------------+------------+
+
+.. |check| raw:: html
+
+   &#x2713;
+
+.. |cross| raw:: html
+
+   &#x2717;
+
+In this article we will provide VM setup instructions assuming the following:
+
+* Ubuntu 20.04 is the Operating System being installed both for the host and VM
+* Setup is done for Intel® Data Center GPU Flex Series
+
+Host BIOS Setup
+---------------
+
+Virtualization support might need to be enabled in host BIOS. Below we provide
+key settings to enable and tune virtualization. Mind that available options and
+their placement depends on the BIOS manufacturer.
+
+For `Intel® Server M50CYP Family <https://ark.intel.com/content/www/us/en/ark/products/series/200321/intel-server-m50cyp-family.html>`_::
+
+    Advanced -> Processor Configuration -> Intel(R) Virtualization Technology: Enabled
+    Advanced -> Integrated IO Configuration -> Intel(R) VT for Directed I/O : Enabled
+    Advanced -> PCI Configuration -> SR-IOV Support : Enabled
+    Advanced -> PCI Configuration -> Memory Mapped I/O above 4GB : Enabled
+    Advanced -> PCI Configuration -> MMIO High Base : 56T
+    Advanced -> PCI Configuration -> Memory Mapped I/O Size : 1024G
+    Advanced -> System Acoustic and Performance Configuration -> Set Fan Profile: Performance
+
+GPU Passthough Virtualization
+-----------------------------
+
+Virtual Machine (VM) setup with GPU passthrough is a type of setup which
+allows exclusive access to GPU from under VM. Mind that with this setup
+only one VM to which GPU was explicitly assigned will be able to use GPU.
+Other VMs and even a host won't have full access to the GPU.
```

Comment:
better to spell out "won't" into "would not"

Reply:
done, but spelled "will not"